CHANGELOG.rst

```diff
@@ -1,3 +1,16 @@
+2.1.0 (2024-02-21)
+------------------
+
+* `#243 <https://github.com/pytest-dev/execnet/pull/243>`__: Added main_thread_only execmodel
+  which is derived from the thread execmodel and only executes ``remote_exec`` calls in the
+  main thread. Callers of ``remote_exec`` must use the returned channel to wait for a task to
+  complete before they call remote_exec again, otherwise the ``remote_exec`` call will fail
+  with a ``concurrent remote_exec would cause deadlock`` error. The main_thread_only execmodel
+  provides solutions for `#96 <https://github.com/pytest-dev/execnet/issues/96>`__ and
+  `pytest-dev/pytest-xdist#620 <https://github.com/pytest-dev/pytest-xdist/issues/620>`__.
+  Also fixed ``init_popen_io`` to use ``closefd=False`` for shared stdin and stdout file
```

Comment:
Also better on its own paragraph. 👍 

```suggestion

  Also fixed ``init_popen_io`` to use ``closefd=False`` for shared stdin and stdout file
```

Reply:
Applied thanks!

Reply:
Hmm, I don't understand what it wants here:
https://results.pre-commit.ci/run/github/97933450/1708640164.sQn3FYJGQKGIhMMK84JS5w
```
Validating...
CHANGELOG.rst:16: D000 Inline emphasis start-string without end-string.
doc/changelog.rst:16: D000 Inline emphasis start-string without end-string.
```

Reply:
I was just missing a space after `*`.